rosweld_drivers/WeldingState params
---